activejob/lib/active_job/exceptions.rb

```diff
@@ -44,14 +44,24 @@ module ClassMethods
       #  end
       def retry_on(*exceptions, wait: 3.seconds, attempts: 5, queue: nil, priority: nil)
         rescue_from(*exceptions) do |error|
+          payload = {
+            job: self,
+            adapter: self.class.queue_adapter,
+            error: error,
+            wait: wait
```

Comment:
Should we add `:wait` to "enqueue_retry.active_job" table in `guides/source/active_support_instrumentation.md`?